internal/pkg/provider/helm/values_test.go

```diff
@@ -0,0 +1,316 @@
+// Copyright 2024 Cofide Limited.
+// SPDX-License-Identifier: Apache-2.0
+
+package helm
+
+import (
+	"testing"
+
+	attestation_policy_proto "github.com/cofide/cofide-api-sdk/gen/go/proto/attestation_policy/v1alpha1"
+	trust_zone_proto "github.com/cofide/cofide-api-sdk/gen/go/proto/trust_zone/v1alpha1"
+	"github.com/cofide/cofidectl/internal/pkg/config"
+	"github.com/cofide/cofidectl/internal/pkg/test/fixtures"
+	"github.com/cofide/cofidectl/pkg/plugin"
+	"github.com/cofide/cofidectl/pkg/plugin/local"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+type Values = map[string]interface{}
+type ValueList = []interface{}
+
+func TestHelmValuesGenerator_GenerateValues_success(t *testing.T) {
+	tests := []struct {
+		name      string
+		trustZone *trust_zone_proto.TrustZone
+		want      Values
+	}{
+		{
+			name: "tz1 no apb or federation",
```

Comment:
Just a small question, is `abp` referring to attestation policy binding?

Reply:
Yes. Is it clear enough or would you prefer something else? binding?